samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/BeginDeliveryActionListener.java

```diff
@@ -296,9 +297,17 @@ private SettingsDeliveryBean populateSettings(PublishedAssessmentIfc pubAssessme
    */
   private void populateDelivery(DeliveryBean delivery, PublishedAssessmentIfc pubAssessment){
 
+    AuthorBean author = (AuthorBean) ContextUtil.lookupBean("author");
     Long publishedAssessmentId = pubAssessment.getPublishedAssessmentId();
     AssessmentAccessControlIfc control = pubAssessment.getAssessmentAccessControl();
     PublishedAssessmentService service = new PublishedAssessmentService();
+    
+    if (author.getIsEditPendingAssessmentFlow()) {
+    	delivery.setRbcAssociation(pubAssessment.getAssessmentId().toString());
+    }
+    else {
+    	delivery.setRbcAssociation(PUBLISHED_ASSESSMENT_RBC_PREFIX + publishedAssessmentId.toString());
```

Comment:
You may as well use the string here, instead of the constant. The constant isn't used anywhere else so doesn't really have any necessity. Me gusta que lo has movado desde la plantilla. Buena idea.